posta/templates/email_nuova_email_servizio.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'email.html' %}
+
+{% block corpo %}
+    <p class="grassetto">Ciao {{ nome }}, benvenuto/a!</p>
+    <p>Il tuo indirizzo e-mail dell'organizzazione è stato attivato.</p>
```

Comment:
Usei "Associazione" (con la maiuscola) per riferirmi a CRI, piuttosto che "organizzazione". @luca-dex, @matthewmazzoleni

Reply:
Meglio ancora sarebbe qualcosa del tipo _la tua casella email di servizio CRI è stata attivata_ . In questa fase sarebbe anche utile fornire dettagli su dove trovare supporto.

Reply:
@luca-dex : per il supporto ho messo il testo che ho visto nelle altre mail di rispondere. Parli anche di doc gsuite ?